### PR TITLE
KF 1.5 OpenShift stack

### PR DESCRIPTION
**Pull request by LaVLaS**
**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**

**Checklist:**
- [ ] Unit tests pass:
  **Make sure you have installed kustomize == 3.2.1**
    1. `make generate-changed-only`
    2. `make test`

### CODE REVIEW

File: openshift/kfdef/kfctl_openshift_v1.5.0_odh.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    name: katib
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/mpi-job
+    name: mpi-job
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/kfserving
```
Comment (yhwang) [inline openshift/kfdef/kfctl_openshift_v1.5.0_odh.yaml:73]: for IKS, we move to KServe. will you do the same change?

**Comment by tks-ai** [comment]
thanks for your nice work. very glad to see this is in progress.
how is it going now?

**Comment by joeswashington** [comment]
Hey @LaVLaS we were interested in seeing this PR moved to merge. How can we help?

File: openshift/kfdef/kfctl_openshift_v1.5.0_odh.yaml (hunk 1 above)
Comment (LaVLaS) [reply openshift/kfdef/kfctl_openshift_v1.5.0_odh.yaml:68]: Apologies for the delay. @VaishnaviHire Is finishing up the work to include kserve as part of this deployment

**Review by crobby — APPROVED** [review]

**Pull request merged**